doc/source/whatsnew/v1.5.0.rst

```diff
@@ -46,7 +46,8 @@ Other enhancements
 - Improved error message in :class:`~pandas.core.window.Rolling` when ``window`` is a frequency and ``NaT`` is in the rolling axis (:issue:`46087`)
 - :class:`Series` and :class:`DataFrame` with ``IntegerDtype`` now supports bitwise operations (:issue:`34463`)
 - Add ``milliseconds`` field support for :class:`~pandas.DateOffset` (:issue:`43371`)
--
+- :meth:`DataFrame.reset_index` now accepts a ``names`` argument which renames the index names (:issue:`6878`)
+- :class:`Index` now contains ``_get_default_index_names`` method to get names of index for DataFrame (:issue:`6878`)
```

Comment:
remove this, we don't advertise / comment on internal functions

Reply:
ok. Noted